### ISSUE TITLE
Felendren the Banished Can't be attacked 

### ISSUE THREAD

**Issue by iwantstofu** [dalaranwow/dalaran-wow#4758]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest in the starting area of the bloodelves "Felendren the Banished," Felendren kept evading attacks.and did not reset.
**Expected behaviour**: Tell us what should happen instead
Felendren should be able to be killed and reset 
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by ashido96** [comment]
ARGH, no links, no pictures, no screenshots, finish your report please.

**Comment by iwantstofu** [comment]
My bad dog ill do it when i get home from work

Sent from my iPhone

On Jun 21, 2017, at 3:25 PM, Kyle Stulen <notifications@github.com<mailto:notifications@github.com>> wrote:


ARGH, no links, no pictures, no screenshots, finish your report please.

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1954#issuecomment-307380950
